pkg/schedule/schedulers/evict_leader.go

```diff
@@ -408,6 +415,7 @@ func (handler *evictLeaderHandler) updateConfig(w http.ResponseWriter, r *http.R
 	batchFloat, ok := input["batch"].(float64)
 	if ok {
 		if batchFloat < 1 || batchFloat > 10 {
+			handler.config.removeStore(id)
```

Comment:
1. Do we allow the input to not contain the store id?
2. How about put L407(pauseLeaderTransferIfStoreNotExist) behind all checks over?

Reply:
1. Yes, we allow only changing batch size.
2. range relies on `exist`, only batch can be moved, but no much difference I think.

Reply:
How about adding a defer after L411? For now, it is so easy to forget to `removeStore`.

```go
defer func () {
    if err != nil {
         ....
    }
}()
```